### PR TITLE
docs(guide/Components): should reference Angular 2

### PR DESCRIPTION
**Pull request by thomasdc**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
docs update

### CODE REVIEW

**Comment by thomasdc** [comment]
I signed it!

File: docs/content/guide/component.ngdoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ Advantages of Components:
 - simpler configuration than plain directives
 - promote sane defaults and best practices
 - optimized for component-based architecture
-- writing component directives will make it easier to upgrade to Angular
+- writing component directives will make it easier to upgrade to Angular 2
```
Comment (gkalpak) [inline docs/content/guide/component.ngdoc]: "Angular" is used deliberately in order to keep in-line with our [branding guidelines][1].

[1]: http://angularjs.blogspot.gr/2017/01/branding-guidelines-for-angular-and.html

**Comment by gkalpak** [comment]
Thx for PR, but the current wording is deliberate (see https://github.com/angular/angular.js/pull/15669/files#r99082265).

File: docs/content/guide/component.ngdoc (hunk 1 above)
Comment (thomasdc) [inline docs/content/guide/component.ngdoc]: Ok, nice to know. Thanks for sharing.

**Pull request closed without merging**